cmd/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gookit/color"
+	"os"
+)
+
+//go:generate go run gen.go
```

Comment:
Is this comment documenting main?

Reply:
Yeah, this was pretty new to me. If you add a comment starting with `//go:generate` then running the `go generate` command will actually scan your code for these comments and execute them. It's usually used to generate new code files during a build. [More details here if you're interested](https://blog.golang.org/generate).